samples/samples/autocommit.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import argparse
+
+from google.cloud.spanner_dbapi import connect
+
+
+def enable_autocommit_mode(instance_id, database_id):
+    """Enables autocommit mode."""
+    # [START enable_autocommit_mode]
+    connection = connect(instance_id, database_id)
+    connection.autocommit = True
+    print("Autocommit mode is enabled.")
+
+    cursor = connection.cursor()
+
+    cursor.execute(
+        """CREATE TABLE Singers (
+                SingerId     INT64 NOT NULL,
+                FirstName    STRING(1024),
+                LastName     STRING(1024),
+                SingerInfo   BYTES(MAX)
+            ) PRIMARY KEY (SingerId)"""
+    )
+
+    cursor.execute(
+        """INSERT INTO Singers (SingerId, FirstName, LastName) VALUES
+            (12, 'Melissa', 'Garcia'),
+            (13, 'Russell', 'Morales'),
+            (14, 'Jacqueline', 'Long'),
+            (15, 'Dylan', 'Shaw')"""
+    )
+
+    cursor.execute("""SELECT * FROM Singers WHERE SingerId = 13""")
+
+    print(
+        u"SingerId: {}, AlbumId: {}, AlbumTitle: {}".format(*cursor.fetchone())
+    )
+
+    connection.close()
+    # [END enable_autocommit_mode]
+
+
+if __name__ == "__main__":
```

Comment:
Should this file be executable?

Reply:
I just meant to ask whether these files should have the `x` bit set so you can run them as `./samples/samples/autocommit.py` instead of `python ...`, but I see the python-spanner samples don't.